urllib3/poolmanager.py

```diff
@@ -110,8 +116,14 @@ class PoolManager(RequestMethods):
 
     proxy = None
 
-    def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
+    def __init__(self, num_pools=10, headers=None, transport_security_manager=None,
+                 **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+
+        if transport_security_manager is None:
+            transport_security_manager = TransportSecurityManager()
```

Comment:
I think that could work as long as the conditional is `if transport_security_manager is True` so we're not accidentally catching "truthy" objects. I'd wait for @Lukasa or @haikuginger to weigh in before making this change though.